src/runtime/recipe/slot-connection.ts

```diff
@@ -180,9 +180,9 @@ export class SlotConnection implements Comparable<SlotConnection> {
     consumeRes.push(`${this.name}:`);
     consumeRes.push('consumes');
     if (this.targetSlot) {
-      consumeRes.push(`${
+      consumeRes.push(
```

Comment:
this would eventually use builder as well, right?

Reply:
I'm not sure this is worth it. If you're just building a single line, there's no need to worry about indentation, and using `string[]` with `join()` is just as ergonomic, and more conventional.